src/UI/Component/Panel/Factory.php

```diff
@@ -79,6 +79,9 @@ public function sub($title,$content);
 	 *   rivals:
 	 *      Standard Panels: >
 	 *        The Report Panels contains sub panels used to structure information.
+	 *      Presentation Table: >
+	 *        Presentation Tables display only a subset of the data at first glance;
+	 *        their entries can then be expanded to show detailed information.
```

Comment:
Could we replace the term "Presentation X" with "Collapsable X" if EXPANDING is the defining property about it?

Reply:
In general I'm not in love with the name, but I prefer naming our components after their purpose, not how they work. I imagine that the first one could stay stable, even if the second one changes.